cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -82,4 +93,36 @@ <h4 class="u-mb5">
     </li>
     {% endfor %}
 </ul>
+
+{% if settings.view_more %}
+  {{ _view_more(settings.view_more) }}
+{% endif %}
+
+{% endmacro %}
+
+{# ==========================================================================
+
+   _view_more()
```

Comment:
If we're going to break this out to it's own macro, we should probably make it globally available so other macros can use it too.